client/material/tooltip.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react'
+import { useTransition, UseTransitionProps } from 'react-spring'
+import styled from 'styled-components'
+import { background100 } from '../styles/colors'
+import { caption } from '../styles/typography'
+import { OriginX, OriginY, PopoverContent, useAnchorPosition } from './popover'
+import { Portal } from './portal'
+import { defaultSpring } from './springs'
+
+const isDev = __WEBPACK_ENV.NODE_ENV !== 'production'
+
+const Container = styled.div`
+  ${caption};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  min-height: 24px;
+  padding: 4px 8px;
+  border-radius: 4px;
+  background-color: ${background100};
```

Comment:
So first, your comment is a lie about this being the "lightest", there is a `background50` :)

But, primary text color doesn't come anywhere close to passing contrast on anything lighter than `background400` so this is a no-go. You can pass contrast by flipping the text to invert (e.g. black 87%) but it feels pretty odd to have black text in general imo.

I would try for some other visual treatment, darker colors with a light border maybe? And I would recommend getting the Contrast plugin in Figma and using it to check these things so that I don't have to check everything myself (Chrome can also tell you in devtools, but it's a bit hard for this particular element since it disappears)